### PR TITLE
Show left area

### LINKED ISSUE #3393: Keyboard shortcut/menu item to hide/show L/R sidepanels
@ian-r-rose 

We should probably have menu items in the View menu for the following:

* Hide/Show the L side panel
* Hide/Show the R side panel

along with keyboard shortcuts. In discussion with @jasongrout & Bloomberg UX, it was felt that a dedicated shortcut to hide/show was better than one that just collapses.

### PR DESCRIPTION
**Pull request by ian-r-rose**
Fixes #3393.

### CODE REVIEW

**Comment by jasongrout** [comment]
FYI, visual studio code uses `Cmd B` on macs.

**Comment by ian-r-rose** [comment]
Thanks, Jason. I have provisionally added that shortcut.

File: packages/application/src/shell.ts [hunk 1]
```diff
@@ -844,6 +897,19 @@ namespace Private {
...
+     * Expand the sidebar.
+     *
+     * #### Notes
+     * This will open the most recently used tab, or the first tab
+     * if there is no most recently used.
+     */
+    expand(): void {
+      const previous =
+        this._lastCurrent || (this._items.length > 0 && this._items[0].widget);
+      this._sideBar.currentTitle = previous.title;
```
Comment (blink1073) [inline packages/application/src/shell.ts]: Previous could be null here

Comment (ian-r-rose) [reply packages/application/src/shell.ts]: Thanks, fixed.

**Comment by ellisonbg** [comment]
Does this still show the L/R tab bar when collapsed to enable mouse driven showing?

**Comment by ian-r-rose** [comment]
@ellisonbg I'm not sure I understand your question. At the moment it hides/shows the L/R tab bar via `Accel B` or via mouse activation of the menu item. When the tab bar is collapsed, it shows the most recently used tab. When the bar is not collapsed, it collapses its.

**Pull request merged**